.github/workflows/pytest-core-nompi.yml

```diff
@@ -31,14 +31,14 @@ jobs:
       matrix:
         name: [
            pytest-ubuntu-py39-gcc11-noomp,
-           pytest-ubuntu-py38-gcc12-omp,
+           pytest-ubuntu-py311-gcc12-omp,
            pytest-ubuntu-py39-gcc7-omp,
            pytest-ubuntu-py310-gcc10-noomp,
            pytest-ubuntu-py38-gcc8-omp,
            pytest-ubuntu-py39-gcc9-omp,
-           pytest-osx-py37-clang-omp,
-           pytest-docker-py37-gcc-omp,
-           pytest-docker-py37-icc-omp,
+           pytest-osx-py38-clang-omp,
+           pytest-docker-py38-gcc-omp,
+           pytest-docker-py38-icc-omp,
            pytest-docker-py38-icx-omp
```

Comment:
If the others are bumped up a version, would it make sense to do the same for this one (3.8->3.9)?

Reply:
I should have another look to "keep some balance for all"